frontend/src/v5/ui/routes/viewer/tickets/newTicket/newTicket.component.tsx

```diff
@@ -30,23 +30,33 @@ import { yupResolver } from '@hookform/resolvers/yup';
 import { TicketsCardHooksSelectors } from '@/v5/services/selectorsHooks';
 import { useContext, useEffect } from 'react';
 import { InputController } from '@controls/inputs/inputController.component';
+import { getWaitablePromise } from '@/v5/helpers/async.helpers';
+import { merge } from 'lodash';
 import { BottomArea, CloseButton, Form, SaveButton } from './newTicket.styles';
 import { TicketForm } from '../ticketsForm/ticketForm.component';
 import { ViewerParams } from '../../../routes.constants';
 import { TicketGroups } from '../ticketsForm/ticketGroups/ticketGroups.component';
 import { TicketContext, TicketDetailsView } from '../ticket.context';
 import { CardContent } from '../ticketsForm/ticketsForm.styles';
 import { TicketsCardViews } from '../tickets.constants';
-import { getWaitablePromise } from '@/v5/helpers/async.helpers';
 
 export const NewTicketCard = () => {
 	const { teamspace, project, containerOrFederation } = useParams<ViewerParams>();
+	const unsavedTicket = TicketsCardHooksSelectors.selectUnsavedTicket();
+	const cardView = TicketsCardHooksSelectors.selectView();
 
 	const isFederation = modelIsFederation(containerOrFederation);
 	const template = TicketsCardHooksSelectors.selectSelectedTemplate();
 	const isLoading = !('config' in template);
+	const templateId = template._id;
 
-	const defaultTicket = getDefaultTicket(template);
+	let defaultTicket = getDefaultTicket(template);
+	if (
+		unsavedTicket?.containerOrFederation === containerOrFederation
+		&& unsavedTicket?.templateId === templateId
+	) {
+		defaultTicket = merge(defaultTicket, unsavedTicket.data);
+	}
```

Comment:
The data still persists after I explicitly cancelled the edition

### Steps to replicate
- I fill in a form to create a ticket
- I click on X
- I click on create with the same template as before

### Current Behaviour
- The previously filled ticket appears again

### Expected Behaviour
- The form should get blanked

### Pictures/ Gifs of the error
![over presistence](https://github.com/3drepo/3drepo.io/assets/42032433/ce0ad03d-f9b9-490a-9e29-a955a79cf5d6)